src/EnergyPlus/Photovoltaics.cc

```diff
@@ -246,6 +247,7 @@ namespace Photovoltaics {
         // Using/Aliasing
         using namespace DataHeatBalance;
 
+        using PhotovoltaicThermalCollectors::GetPVTmodelIndex;
```

Comment:
We try to get rid of using statements.  It's not a red flag, but this is just one more that we have to remove later.